profiles/arch/arm/armv6j/use.force

```diff
@@ -0,0 +1,10 @@
+# Force armv6 specific USE flags
+cpu_flags_arm_v4
+cpu_flags_arm_v5
+cpu_flags_arm_v6
+
+cpu_flags_arm_edsp
+cpu_flags_arm_thumb
+cpu_flags_arm_thumb2
+cpu_flags_arm_vfp
```

Comment:
https://www.arm.com/products/processors/technologies/vector-floating-point.php

VFPv2 is an optional extension to the ARM instruction set in the ARMv5TE, ARMv5TEJ and ARMv6 architectures.

Being optional it shouldnt be forced